numpy/_core/_ufunc_config.pyi

```diff
@@ -1,10 +1,12 @@
+from _typeshed import SupportsWrite
 from collections.abc import Callable
 from typing import Any, Literal, TypeAlias, TypedDict, type_check_only
 
-from numpy import _SupportsWrite
+from numpy import errstate as errstate
```

Comment:
I'm curious why you cannot just import errstate?

Reply:
Such a `import {} as {}` is an "explicit re-export". If there'd have been an `__all__` (which there should be, but seemed out of scope to me for this PR), adding `"errstate"` to it would've achieved the same result.